src/js/plugin-bridge/PluginSDK.js

```diff
@@ -264,7 +267,7 @@ const getSDK = function (pluginID, config) {
     addStoreConfig,
     dispatch: createDispatcher(pluginID),
     Store: StoreAPI,
-    Hooks,
+    Hooks: hooks,
```

Comment:
Yeah this file is most likely going to disappear with our new efforts so wouldn't worry :)